Added optional params subject_text and body_text…

knifhen opened the pull request:
…  that can be used instead of reading subject and body from files

Commit:
Added optional subject_text and body_text that can be used instead of reading subject and body from files

mdomke commented on README.md:
```diff
@@ -47,7 +47,9 @@ Sending an email message:
 #### Parameters
 
 * `to`: _(mandatory)_ A list of recipients as strings.
+* `subject_text`: _(optional)_ Plain text email subject, instead of `subject`.
```
Could you make clear that either the `subject_text` _or_ the `subject` option have to be supplied. The same applies for the `body` and the `body_text`.

mdomke commented on opt/resource/out:
```diff
@@ -67,22 +67,30 @@ def get_mime_subtype(payload):
+    if not subject_text:
+        fname = get_value_from_payload(payload, 'params', 'subject')
+        with open(os.path.join(src, fname)) as fp:
+            return Template(fp.read()).render(template_vars)
```
I would rather use a variable `subject` for both config values `'subject_text'` and `'subject'` and assign its value accordingly. Additionally I think that the template variables should be applied in any case.

``` python
    subject = get_value_from_payload(payload, 'params', 'subject_text', optional=True)
    if not subject:
        fname = get_value_from_payload(payload, 'params', 'subject')
        with open(os.path.join(src, fname)) as fp:
            subject = fp.read()
    return Template(subject).render(template_vars)
```

mdomke commented on opt/resource/out:
```diff
@@ -67,22 +67,30 @@ def get_mime_subtype(payload):
-    inline = get_value_from_payload(payload, 'params', 'inline_css', default=True, optional=True)
-    mime_subtype = get_mime_subtype(payload)
+    body_text = get_value_from_payload(payload, 'params', 'body_text', optional=True)
+    if not body_text:
```
As described above. The body should be read either from the constant text in `'body_text'` or from the file `'body'`. All other operations (e.g: css inlining, template variable application) should be applied regardless of the source of the body-text.

``` python
body = get_value_from_payload(payload, 'params', 'body_text', optional=True)
if not body:
    fname = get_value_from_payload(payload, 'params', 'body')
    with open(os.path.join(src, fname)) as fp:
        message = fp.read()
message = Template(message).render(template_vars)
...
if mime_subtype == 'html' and inline:
    ...
return message
```

mdomke reviewed (changes requested):
Hi @knifhen! Thanks for your pull-request! I appreciate your contribution very much! Nevertheless I think that some things should be handled a little different from the way you implemented it. It would be great if you could adjust your pull-request accordingly or give some feedback on why you think it should be as it is.

Commit:
Render template when using subject_text param, same as when reading subject from file

Apply inline css and template when using body_text param, same as when reading body message from file

Commit:
Trying to be more clear about how to use subject/subject_text and body/body_text

knifhen commented:
Hi @mdomke, I tried to make the pull request more to your specification. That was some good points you made about the change. Hope this works better!

Regards Andreas Knifh
